### ISSUE TITLE
[Mono] Compiling project crashes editor, but project still runs (mscorlib.dll.dll)

### ISSUE THREAD

**Issue by ax95** [godotengine/godot#39249]
**Godot version:**
Windows 64 Mono, occurs both in v3.2.2 Beta 3 and v.3.2.1 stable.

**OS/device including version:**
Windows 8.1.

**Issue description:**
This started occurring to me only after I exported a project for Windows for the first time, but persists across different projects.

When running any project through Visual Studio Code, or running through the Godot editor – if the project needs to be compiled (there are any changes made to the c# scripts within), the Godot editor instantly crashes. However, the Godot .exe still runs my game (shows up as a window). This occurs as stated even when building through VSCode with the editor in the background.

Attached is one of several logs produced when this happened from mono complaining about trying to use mscorlib.dll.dll. The mscorlib.dll exists within `GodotSharp\Mono\lib\mono\4.5` as it should. Renaming it to mscorlib.dll.dll crashes the Godot launcher immediately. Having mscorlib.dll and a copy of the file renamed to "mscorlib.dll.dll" does the same thing.

[2020_06_03 01.07.53 (50192).txt](https://github.com/godotengine/godot/files/4720112/2020_06_03.01.07.53.50192.txt)

Again, the build appears to actually succeed, as the compiled project show whatever minor change I made to its code. Nevertheless, the editor will still crash with no exceptions. Running the launcher through cmd does not produce any further errors.

**Steps to reproduce / minimal reproduction project:**
I highly doubt the problem is reproducible, as all I did before and after this problem started occurring was exporting my project for windows. I recently created a new empty project, and the same issue occurs with it.

**Comment by ax95** [comment]
It has now seems as if building from within the Godot editor only __sometimes__ crashes, but it's still the same deal. Nothing about it seems to make sense. Here's the two last logs that someone better than me might be able to decipher:

[2020_06_03 06.27.06 (136444).txt](https://github.com/godotengine/godot/files/4721045/2020_06_03.06.27.06.136444.txt)

[2020_06_03 06.27.55 (73252).txt](https://github.com/godotengine/godot/files/4721046/2020_06_03.06.27.55.73252.txt)

**Comment by ax95** [comment]
Yep. I just now compiled the project 4 times from VSCode, with a code change each time, and only the last time did the Godot Editor crash. Let me reiterate: this doesn't make sense. Last log, once again, complaining about a missing mscorlib.dll.dll.

However, since the last few builds didn't crash the editor, and they also contained the same complaints, it doesn't seem like the "mscorlib.dll.dll" thing is the issue here. I would like to figure out what _is_, however.

**Comment by neikeq** [comment]
I don't see anything wrong in the mono logs. Anything else that could help debug this would be welcome. Can you provide an example project and a list of steps to follow to reproduce the bug? If that's not possible you could try running Godot with a debugger to find the crash backtrace, but I don't know how helpful the debugger information will be if it's not a debug build.

**Comment by ax95** [comment]
Hi, I'm choosing to close this issue as variables unknown to me have since made themselves clear within my operating system (trojan, rootkit), and I expect this issue to be purely on my end and therefore fixed after a re-installation of my os. Should the problem persist, I will reopen a new issue with clearer and more debugging information.
